### PR TITLE
 [SHMEM 1.6 Sec 9.10] Collectives section committee changes

### PR DESCRIPTION
**Pull request by davidozog**
# Summary of changes
This includes these PRs:
https://github.com/davidozog/openshmem-specification/pull/11
https://github.com/davidozog/openshmem-specification/pull/10
https://github.com/davidozog/openshmem-specification/pull/9
https://github.com/davidozog/openshmem-specification/pull/8

Any corresponding changelog entries are in the backmatter section.

This PR about reduction associativity is not included and will likely will not make it for RC1:
https://github.com/davidozog/openshmem-specification/pull/12

# Proposal Checklist
- [X] Link to issue(s)
- [ ] Changelog entry
- [X] Reviewed for changes to front matter
- [X] Reviewed for changes to back matter

### CODE REVIEW

File: content/programming_model_overview.tex [hunk 1]
```diff
@@ -144,7 +144,7 @@
       data object on another symmetric data object.
   \item \OPR{All-to-All}: All \acp{PE} participating in the routine exchange
       a fixed amount of contiguous or strided data with all other \acp{PE}
-      in the active set.
+      in the team.
```
Comment (jdinan) [inline content/programming_model_overview.tex:147]: @davidozog @kwaters4 This line was also updated by the front matter section committee. I am going to drop this change from the draft as the front matter committee change is more consistent across this section. Please review and let me know if you have any concerns.

Comment (jdinan) [reply content/programming_model_overview.tex:147]: I had to rewrite history a bit to make the merges go seamlessly. This is my version of the collectives branch that dropped the above change: https://github.com/openshmem-org/specification/compare/master...jdinan:specification:collectives_section-v1.6-RC1

Comment (davidozog) [reply content/programming_model_overview.tex:147]: Thanks @jdinan - sorry for the conflict.

File: content/shmem_alltoall.tex [hunk 2]
```diff
@@ -116,23 +133,26 @@
...
 
     Upon return from a \FUNC{shmem\_alltoall} routine, the following is true for
     the local PE:
     \begin{itemize}
-    \item Its \VAR{dest} symmetric data object is completely updated and
-    the data has been copied out of the \VAR{source} data object.
-    \item For active-set-based routines,
-    the values in the \VAR{pSync} array are restored to the original values.
+    	\item Its \VAR{dest} symmetric data object is completely updated and the 
+	data has been copied out of the source data object.
```
Comment (davidozog) [inline content/shmem_alltoall.tex:151]:
```suggestion
	data has been copied out of the \VAR{source} data object.
```

File: content/shmem_broadcast.tex [hunk 3]
```diff
@@ -75,66 +74,92 @@
...
     For active-set-based broadcasts:
     \begin{itemize}
-    \item The \dest{} object is updated on all \acp{PE} other than the
-      root \ac{PE}.
-    \item All \acp{PE} in the active set defined by the
-      \VAR{PE\_start}, \VAR{logPE\_stride}, \VAR{PE\_size} triplet
-      must participate in the operation.
-    \item Only \acp{PE} in the active set may call the routine.  If a
-      \ac{PE} not in the active set calls an active-set-based
+        \item The \VAR{dest} object is updated on all PEs other than the root PE.
```
Comment (davidozog) [inline content/shmem_broadcast.tex:117]:
```suggestion
        \item The \VAR{dest} object is updated on all \acp{PE} other than the root PE.
```

File: content/shmem_collect.tex [hunk 4]
```diff
@@ -66,15 +66,13 @@
     \openshmem \FUNC{collect} and \FUNC{fcollect} routines perform a collective
     operation to concatenate \VAR{nelems}
     data items from the \source{} array into the
-    \dest{} array, over an \openshmem team or active set
-    in processor number order. The resultant \dest{} array contains the contribution from
+    \dest{} array, over an \openshmem team in processor number order.
```
Comment (davidozog) [inline content/shmem_collect.tex:69]:
```suggestion
    \dest{} array, over an \openshmem team in process number order. 
```

File: content/shmem_sync.tex [hunk 5]
```diff
@@ -1,7 +1,11 @@
 \apisummary{
     Registers the arrival of a \ac{PE} at a synchronization point.
     This routine does not return until all other \acp{PE} in a given OpenSHMEM team
-    or active set arrive at this synchronization point.
+    arrive at this synchronization point.
```
Comment (davidozog) [inline content/shmem_sync.tex:4]:
```suggestion
    arrives at this synchronization point.
```

**Comment by davidozog** [comment]
@jdinan - this PR can be closed, correct?

I double checked all the section changes into RC1 and edits from the reading are accounted for.

**Pull request closed without merging**